<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络调试助手 1.5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconfile.qrc">
    <normaloff>:/images/network.ico</normaloff>:/images/network.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>网络设置</string>
    </property>
    <widget class="QComboBox" name="comboBox_type">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>UDP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_type">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>1.协议类型</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>2.本地主机地址</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ip">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_port">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>3.本地主机端口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>180</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>打开</string>
     </property>
     <property name="icon">
      <iconset resource="iconfile.qrc">
       <normaloff>:/images/ledoff.png</normaloff>
       <normalon>:/images/ledon.png</normalon>:/images/ledoff.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>接收设置</string>
    </property>
    <widget class="QRadioButton" name="radioButton_recv_ascii">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>119</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_recv_hex">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>119</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>发送设置</string>
    </property>
    <widget class="QRadioButton" name="radioButton_send_ascii">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>119</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>ASCII</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_send_hex">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>119</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>HEX</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_send_clear">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清除发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>571</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>接收数据</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_recv">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>425</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>455</width>
      <height>141</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>发送数据</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit_send">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>455</width>
        <height>120</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_send">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>480</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_remote">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>311</width>
      <height>25</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboBox_remote">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_remote">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>3</y>
       <width>69</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>远程主机</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_recv_clear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>清除接收</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="iconfile.qrc"/>
 </resources>
 <connections/>
</ui>
